components/sidebar/sidebar.jsx

```diff
@@ -35,7 +35,6 @@ export default class Sidebar extends Component {
           height={ availableHeight } />
 
         <div className="sidebar__inner">
-          <img src="https://img.shields.io/badge/webpack-v2.3.3-blue.svg" />
```

Comment:
Leave it for now. You still want to show the version globally I think and sidebar is a nice place for that.